src/__tests__/consent-manager-builder/index.todo.js

```diff
@@ -169,6 +170,156 @@ describe('ConsentManagerBuilder', () => {
     )
   })
 
+  test('if defaultDestinationBehavior is set to imply and category is set to true, loads new destination', done => {
+    document.cookie =
+      'tracking-preferences={%22version%22:1%2C%22destinations%22:{%22Amplitude%22:true}%2C%22custom%22:{%22advertising%22:false%2C%22marketingAndAnalytics%22:true%2C%22functional%22:true}}'
+    window.analytics = { load() {}, identify() {} }
+
+    console.log('document.cookie ', document.cookie)
```

Comment:
Intentionally logging this, or good to remove?

Reply:
Good to remove 😄